### PR TITLE
[Docs] Add Wallets page to Getting Started docs

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

Adds a new Wallets page

### Tested

Using Markdown preview renderer locally

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/docs/getting-started/wallets.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Digital Wallets for Celo
+
+## Choosing a Wallet
+
+Wallets are tools that create accounts, manage keys, and help users transact on the Celo network.
+
+It's important to be careful when choosing a wallet because they manage your secret account keys. You should only use reputable wallets that are well maintained by organizations/people that you trust.
+
+The [Celo Native Wallets](#celo-native-wallets) section shows some popular wallets that were built specifically for the Celo network. They often includes features that more general wallets do not, like paying for fees with cUSD. The [Celo Compatible Wallets](#celo-compatible-wallets) section has wallets that can work with Celo but were built for other networks (like Ethereum) or through company partnerships (like Pesabase).
```
Comment (critesjosh) [inline packages/docs/getting-started/wallets.md]:
```suggestion
The [Celo Native Wallets](#celo-native-wallets) section shows some popular wallets that were built specifically for the Celo network. They often include features that more general wallets do not, like paying for fees with cUSD. The [Celo Compatible Wallets](#celo-compatible-wallets) section has wallets that can work with Celo but were built for other networks (like Ethereum) or through company partnerships (like Pesabase).
```

**Review by critesjosh — APPROVED** [review]
Minor spelling update, but looks good.

So Nifty and Dove will support Celo after the donut hardfork? (not just compatibility)

**Comment by jmrossy** [comment]
> So Nifty and Dove will support Celo after the donut hardfork? (not just compatibility)

@critesjosh My understanding is they support it already, that's why I included them here. Do I need to rephrase to make that more clear?

**Comment by critesjosh** [comment]
> > So Nifty and Dove will support Celo after the donut hardfork? (not just compatibility)
> 
> @critesjosh My understanding is they support it already, that's why I included them here. Do I need to rephrase to make that more clear?

I just tried using the Nifty extension wallet. It doesn't really work with Celo right now. It still references ETH and I tried sending a transaction, but it failed. I assume because it tried to sign an ETH transaction rather than a Celo transaction.

Screen Shot 2021-04-05 at 12 36 13 PM

I tried the Dove wallet and it supports Celo and I believe pesabase does as well.

After the donut hardfork we should add some more extensive docs around using ETH wallets with Celo.

**Pull request merged**